polkadot-protocol/onchain-governance/overview.md

```diff
@@ -1,4 +1,326 @@
 ---
 title: Overview
 description: TODO
```

Comment:
missing description

@kapetan3sid wdyt about this one? seems like a CTA to me haha but maybe it works..

```suggestion
description: Explore Polkadot’s advanced governance model, OpenGov, where direct democracy, flexible voting options, and stakeholder participation drive network evolution.
```